src/routes/ViewAllLogs/ViewAllLogs.js

```diff
@@ -58,8 +58,7 @@ const visibleColumns = [
   'hrId',
 ];
 
-const INITIAL_RESULT_COUNT = 25;
```

Comment:
Keep this const, please, even if it has the same value as `RESULT_COUNT_INCREMENT`, it has a different meaning. If you search through the project, you will see that this approach is used in many places.

Reply:
Done